src/tag/index.js

```diff
@@ -41,23 +41,22 @@ class Tag extends PureComponent {
     const { id, label, labelRemove = 'Remove', readOnly, disabled } = this.props
 
     const tagId = `${id}_tag`
+    const buttonId = `${id}_button`
     const className = cx('tag-remove', { readOnly }, { disabled })
     const isDisabled = readOnly || disabled
-    const onClick = !isDisabled ? this.handleClick : undefined
-    const onKeyDown = !isDisabled ? this.onKeyDown : undefined
-    const onKeyUp = !isDisabled ? this.onKeyUp : undefined
 
     return (
-      <span className={cx('tag')} id={tagId}>
-        {label}
+      <span className={cx('tag')}>
+        <span id={tagId}>{label}</span>
```

Comment:
This could be potentially a breaking change as it alters the HTML markup rendered by previous versions.

Reply:
I thought it to be very unlikely that someone would have any selectors affected by this change. I weighted it against setting aria-label={label} which also solves the problem with ”remove” being spoken when focusing the trigger node.

Reply:
Yeah, I'm not against changing the HTML but then it'd be a major version bump (to be safe, just in case anyone is relying on it). I think, setting `aria-label={label}` is probably preferable.